pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -8,18 +8,17 @@
 { autoPatchelfHook
 , blas
 , cmake
-, cudaPackages
 , cudaSupport ? config.cudaSupport
 , fetchurl
 , gfortran
-, cudaCapabilities ? cudaPackages.cudaFlags.cudaCapabilities
 , gpuTargets ? [ ] # Non-CUDA targets, that is HIP
 , rocmPackages
 , lapack
 , lib
 , libpthreadstubs
 , magmaRelease
 , ninja
+, pkgs
```

Comment:
Can ask for `cudaPackages_11`

Reply:
I'd prefer to instead conditionally access it instead. It might just be scar tissue from the frequent problem I ran into under cross compilation where `callPackage` would error if an argument wasn't available, even though it was unused.

Thoughts?

Reply:
`cudaPackages_11 ? null`

Reply:
This way it stays overridable even without overlays, and the "input" to callPackage is strictly smaller (in the sense that pkgs is as big as it gets)

Reply:
Oh! Good point!